File: src/main/java/org/polyfrost/polytime/command/TimeCommand.java
package org.polyfrost.polytime.command;

import org.polyfrost.polytime.PolyTime;
import cc.polyfrost.oneconfig.utils.commands.annotations.Command;
import cc.polyfrost.oneconfig.utils.commands.annotations.Main;

@Command(value = PolyTime.MODID, description = "Access the " + PolyTime.NAME + " GUI.", aliases = "timechanger")
public class TimeCommand {

    @Main
    private void main() {
        PolyTime.INSTANCE.config.openGui();
    }
}

File: src/main/kotlin/org/polyfrost/polytime/command/TimeCommand.kt
package org.polyfrost.polytime.command

import org.polyfrost.oneconfig.api.commands.v1.factories.annotated.Command
import org.polyfrost.polytime.config.ModConfig
import org.polyfrost.polytime.PolyTime

@Command(PolyTime.MODID, "timechanger")
object TimeCommand {
    @Command
    private fun main() {
        // TODO
        // ModConfig.openGui()
    }
}